src/mlpack/methods/decision_tree/gini_gain.hpp

```diff
@@ -88,17 +89,23 @@ class GiniGain
 
       // SIMD loop: add counts for four elements simultaneously (if the compiler
       // manages to vectorize the loop).
+      // #pragma omp parallel for reduction (+:accWeights)
       for (size_t i = 3; i < labels.n_elem; i += 4)
       {
         const double weight1 = weights[i - 3];
         const double weight2 = weights[i - 2];
         const double weight3 = weights[i - 1];
         const double weight4 = weights[i];
 
+        // #pragma omp atomic
```

Comment:
Hi @Param-29, I don't think there should be any error associated with this and can be un-commented.

Reply:
I removed #pragma omp atomic because its presence doesnot matter as it is not in parallel section of the code.